imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -718,7 +722,7 @@ private File localCdrDir() {
     //          while collecting a list of errors on the go.
     //          Does not proceed to phase 2 if any error occurred and returns an error
     @Override //NO CHECKSTYLE Cyclomatic Complexity
-    public void cdrProcessingPhase1(CdrFileNotificationRequest request) {
+    public void cdrProcessPhase1(CdrFileNotificationRequest request) {
```

Comment:
Random inclusion: made it consistent with all the other cdrProcessPhaseX methods